packages/FormSync/FormSyncElement.js

```diff
@@ -478,11 +496,25 @@
                 return;
             }
             const sync = new InputSync();
+
+            const isSelectOptionClonedToInput = originalInput.tagName === 'OPTION' && originalInput.tagName !== clonedInput.tagName;
```

Comment:
Ich würde die Funktion auslagern in etwas wie `checkCompatibilityOfOriginalAndClonedElements`, damit `syncInput` möglichst clean und präzise bleibt.